src/content_sao.cpp

```diff
@@ -477,7 +483,9 @@ void LuaEntitySAO::step(float dtime, bool send_recommended)
 		move_d += m_last_sent_move_precision;
 		float vel_d = m_velocity.getDistanceFrom(m_last_sent_velocity);
 		if (move_d > minchange || vel_d > minchange ||
-				std::fabs(m_yaw - m_last_sent_yaw) > 1.0) {
+				std::fabs(m_yaw - m_last_sent_yaw) ||
```

Comment:
`> 1.0`

Reply:
Not sure what you want here. Should it be `> 1.0f`?

Reply:
Thanks, it's checking if it is true, not greater than. Can't believe I missed that.

Reply:
1.0f

Reply:
I thought so! Floats are being compared not doubles.